### PR TITLE
Center shields.io elements in the README.md file and grammar fixes for multiple documents.

### PR DESCRIPTION
**Pull request by Dwang-ML**
Shields.io elements are now centerized in README.md. 

Also made changes in multiple documents, changes include:
 - Grammar fixes
 - Spelling fixes
 - Rephrasing for clear message delivery and more clarity
 - Remove extra characters, such as blank spaces

Documents with changes mentioned above:
 - README.md 
 - CHANGELOG.md
 - CODE_OF_CONDUCT.md
 - CONTRIBUTING.md
 - ISSUE_TEMPLATE.md
 - RELEASE_CHECKLIST.md
 - doc/about/faq.rst
 - doc/about/intro.rst
 
Note: I may be missing some documents in the list above. I apologize if I am.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Examples of unacceptable behavior by participants include:
 * Public or private harassment
 * Publishing others' private information, such as a physical or electronic
   address, without explicit permission
-* Other conduct which could reasonably be considered inappropriate in a
+* Other conducts that could reasonably be considered inappropriate in a
```
Comment (pvcraven) [inline CODE_OF_CONDUCT.md]: The original singular 'conduct' is correct. Grammarly agrees.

File: doc/about/intro.rst [hunk 2]
```diff
@@ -20,10 +20,9 @@ of resources to help you learn.
 Community driven
 ----------------
 
-Arcade is a community driven project. We cannot exist without the help of our users.
-You don't need to be an experienced programmer or a game developer to help us.
-You can help us by reporting bugs, suggesting improvements, pointing out issues
-in documentation or even by sharing your projects with us.
+Arcade is a community-driven project, and we rely on support from users like you.
+You don’t need to be an expert programmer or game developer to contribute.
```
Comment (pvcraven) [inline doc/about/intro.rst]: No curly quotes in readme markdown for this project please. They do get auto-corrected.

**Review by pvcraven — CHANGES_REQUESTED** [review]
Just needs a couple updates then I'm happy merging it in.

**Comment by Dwang-ML** [comment]
Will do, I'll finish today.

File: doc/about/intro.rst (hunk 2 above)
Comment (Dwang-ML) [reply doc/about/intro.rst]: > No curly quotes in readme markdown for this project please. They do get auto-corrected.

I'm confused on what you mean by this 😊

Comment (Dwang-ML) [reply doc/about/intro.rst]: Never mind, I was not observant enough, I apologize.

**Review by pvcraven — APPROVED** [review]

**Comment by pvcraven** [comment]
Thanks!

**Pull request merged**